### ISSUE TITLE
Enhance pbench-move-results to detect the simple case of multiple pbench results in the wrong sub-directory

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#977]
There appear to be cases where 10s of pbench results are being placed into a sub-directory of `/var/lib/pbench-agent` such that `pbench-move-results` considers them to be one big pbench result.

It seems we need to modify the benchmark scripts to be sure they all created `metadata.log` files and then modify `pbench-move-results` to only work on a directory that has a `metadata.log` file in it to avoid this really bad behavior (bad in that multiple gigabytes of data are being gratuitously archived as a pbench result when it is not such an entity).

**Comment by portante** [comment]
We can't use `metadata.log` as the detection because those user invoking start/stop tools directly won't necessarily have that.

**Comment by ndokos** [comment]
That is true, but those results will not be indexed (we depend on metadata.log for that), so a warning is warranted in any case. Not sure that there is a surefire way of detecting it.
